_posts/2017-10-13-jb-in-da-house.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "J.B. Rainsberger in da house"
+date: 2017-10-13 14:03:00 +0200
+categories: learn product 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 3
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-13-jbhc17/poster.jpg
+---
+
+It started in May, that we thought about inviting [J.B. Rainsberger][jbrains] to HolidayCheck. Yeah, there was some bias from my side, since I had the pleasure to have learned from him a couple years ago. So I was all in. I just had to convince my colleagues and the company to pay for it. And it went quite smooth. HolidayCheck is quite committed to improve continuously.
+
+[jbrains]: https://twitter.com/jbrains
+
+## 7:26min
+
+You have never heard of “7 minutes 26 seconds”? Now you did! It is a short and inspiring [lightning talk][7-26-talk] by J.B. called “7 minutes, 26 seconds, and the Fundamental Theorem of Agile Software Development”. Here at HolidayCheck people refer to it as the “Accidental Complication“ talk. I think it inspired many people here to be as excited about having J.B. around. 
+
+It’s not even necessary mentioning that this event won’t be a pure TDD session, or a programmers-only workshop. Even though I had thought of it like that before. Melanie, who is one of our product owners, jumped in early to help to make [#jbhc17][jbhc17] happen. And it became clear right away that this event will not only be teaching us developers, but also the product people about the tools we need to deliver value as fast as possible to our Urlauber (how we call our users).
+
+[7-26-talk]: https://vimeo.com/79106557
+[jbhc17]: https://twitter.com/search?q=%23jbhc17&src=typd
+
+## Remove bottlenecks
+
+The topic “Move the bottleneck out of product development” makes it very clear that our goal is to lead the market. Once we have no bottlenecks in product development. Once we deliver the best thing for our Urlauber. Once we run at high speed. Once we have no useless friction slowing us down, we will be able to lead the market. We will deliver the best value. And we will have even more fun and be the best Urlauber company. And we get where we want to be: we will have to look for the bottlenecks in the market, not in our company anymore. We will be able to try new business models, we will try to innovate, build, measure and learn at high speed.
```

Comment:
`Once we ran out of speed.` (mind the dot at the end) sounds like we did run out of speed at some point in time. How about splitting those 4 sentences with a `,` creating list of things to be done (dependencies) and finally `we will be able to ...`?

But I might be mistaken and this perhaps is a proper way of listing those things out.

Reply:
As you use `will`, I guess you should add it here as well: `And we *will* get where we want`.

Reply:
This whole sentence (`And we get where we want to be: we will have to look for the bottlenecks in the market, not in our company anymore.`) sounds a bit odd:
 - will we **have to** look for bottlenecks in the market? I'd rather look for wide open highways, opportunities :) (or at things like [Kim is doing](http://kimjongunlookingatthings.tumblr.com/) ;) )

The next sentence fits very well - `we will be able to try new business models`.

Reply:
@smolak hehe, this is actually a reference to the book project phoenix, which mentions this. once you have no bottlenecks left in the company because u can move at lightning speed, there is no bottleneck in the company anymore. So it must be external. Which is the market and we, in the end, want to determine and lead the market.

Reply:
OK :)